lib/modules/dosomething/dosomething_api/includes/Transformer.php

```diff
@@ -445,7 +445,8 @@ protected function transformSignup($data, $current) {
    */
   protected function transformUser($data) {
     return [
-      'id' => $data->id,
+      'id' => $data->drupal_id,
```

Comment:
Yep, `$data->id` should already be the NS id, but doesn't really get populated unless the `load_user` query param is added and set to `true`. However, it only really applies to the Reportback endpoint(s). @deadlybutter this shouldn't drastically change your PR I don't think... are you actually using the Drupal ID or were you just adding to differentiate between the two?